src/Templates/Boilerplate/Bit.Boilerplate/src/Client/Boilerplate.Client.Core/Components/Layout/Footer.razor.cs

```diff
@@ -23,7 +24,13 @@ protected override Task OnInitAsync()
 
     private async Task OnCultureChanged()
     {
-        await JSRuntime.SetCookie(".AspNetCore.Culture", $"c={SelectedCulture}|uic={SelectedCulture}", expiresIn: 30 * 24 * 3600, rememberMe: true);
+        await cookie.Set(new ButilCookie
```

Comment:
Use
```csharp
await cookie.Set(new()
...
```
instead

Reply:
yeah you would think, but no, it's not working like this :)